kcidb/monitor/output.py

```diff
@@ -184,9 +185,11 @@ def render(self):
             subject = subject[:NOTIFICATION_MESSAGE_SUBJECT_MAX_LEN - 1] + "✂️"
             LOGGER.warning("Subject is %s characters too long, truncated",
                            subject_extra_characters)
-        assert NOTIFICATION_MESSAGE_SUBJECT_RE.fullmatch(subject), \
-            f"Subject is invalid, must match " \
-            f"{NOTIFICATION_MESSAGE_SUBJECT_RE.pattern} regex"
+        if not NOTIFICATION_MESSAGE_SUBJECT_RE.fullmatch(subject):
```

Comment:
@spbnick 
I am thinking of making this come before truncating the subject. 
What do you think?

Reply:
That would be correct if we were changing the length of the string. However, as I suggest in my review we shouldn't be doing that. Still conceptually it would make better sense to the reader if we did that first, so please go ahead in any case :smile: Thanks!

Reply:
Thanks @spbnick 
Since we are replacing the character I will just leave it there 
I thought we will be removing the unwanted character